src/globus_sdk/authorizers/client_credentials.py

```diff
@@ -75,15 +77,17 @@ def __init__(
 
         super().__init__(access_token, expires_at, on_refresh)
 
-    def _get_token_response(self) -> OAuthTokenResponse:
+    def _get_token_response(self) -> globus_sdk.ClientCredentialsTokenResponse:
         """
-        Make a client credentials grant
+        Make a client credentials request for new tokens.
```

Comment:
```suggestion
        Make a request for new client credentials tokens.
```

Reply:
I actually don't like either of these phrasings.

> a client credentials request

:point_up: what does that mean?

> client credentials tokens

:point_up_2: who dis?

---

Neither of these refer to things in a precise way, and there's no well-known thing called "a client credentials token".

I'm planning to update to
> Make a request for new tokens, using a 'client_credentials' grant.

It might be pedantic, but it's accurate.

Reply:
I accidentally committed this (meant to hit the suggestion below), but I'll just update again "over the top". I'm un-resolving for now, but please re-resolve if you like the result!